internal/driver/driver_test.go

```diff
@@ -1020,34 +1018,221 @@ func TestTagFilter(t *testing.T) {
 	}
 }
 
+func TestIdentifyNumLabelUnits(t *testing.T) {
+	var tagFilterTests = []struct {
+		name               string
```

Comment:
This is typically called "desc" and should be more descriptive than "test1", "test2" and so on. Describe what the actual test case the test covers.